config/mutable.go

```diff
@@ -0,0 +1,97 @@
+// Licensed under the Apache License, Version 2.0
+// Details: https://raw.githubusercontent.com/maniksurtani/quotaservice/master/LICENSE
+
+package config
+
+import (
+	"errors"
+
+	pbconfig "github.com/maniksurtani/quotaservice/protos/config"
+)
+
+func CreateBucket(clonedCfg *pbconfig.ServiceConfig, namespace string, b *pbconfig.BucketConfig) error {
+	if namespace == GlobalNamespace {
+		// TODO(steved) what to do?
```

Comment:
The default is no global default bucket. 

Yes, it should probably fail if trying to create a bucket that already exists.